libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -80,6 +80,25 @@ void StaticAnalyzer::endVisit(FunctionDefinition const&)
 	m_localVarUseCount.clear();
 }
 
+bool StaticAnalyzer::visit(ModifierInvocation const& _modifier)
+{
+	if (m_constructor)
```

Comment:
Can you limit the indentation somehow, for example by returning early here?

Reply:
Sure, I am not a fan of early returns in these visitors because they tend to accumulate different checks and keeping track of invariants across the code flow can be hard.